### ISSUE TITLE
Add support for inject optional header and footer to NotificationDropdown

### ISSUE THREAD

**Issue by sheng-duetto** [GetStream/react-activity-feed#49]
First of all thank you for providing these components they make working with stream much more streamlined.

This is more of a request and not an issue. We have a use case where we need to add some additional links in a header and footer section of the `NotificationDropdown`.  Basically like facebook's dropdown:
![Screen Shot 2019-05-28 at 6 25 24 PM](https://user-images.githubusercontent.com/45951428/58522627-1336e700-8176-11e9-8644-7b06960f5fb3.png)

We can get around this by forking the `NotificationDropdown` component and but would like to see if you guys would be open to add header and footer as an optional Renderable prop since it might be common use case for others.

This way it's much easier for us to pick up updates in the future and not have to main our separate monkey patched version.

E.g. it could be something simple like:
```
diff --git a/src/components/NotificationDropdown.js b/src/components/NotificationDropdown.js
index b5f3a9f..9458b21 100644
--- a/src/components/NotificationDropdown.js
+++ b/src/components/NotificationDropdown.js
@@ -18,6 +18,8 @@ import type {
 } from '../types';
 import type { FeedRequestOptions, FeedResponse } from 'getstream';

+import { smartRender } from '../utils';
+
 type Props = {|
   /** The feed group part of the feed that should be displayed */
   feedGroup: string,
@@ -194,6 +196,7 @@ class NotificationDropdownInner extends React.Component<PropsInner, State> {
         >
           {this.state.open && (
             <DropdownPanel arrow right={this.props.right}>
+              {smartRender(this.props.Header, {}, null)}
               <NotificationFeed
                 feedGroup={this.props.feedGroup}
                 userId={this.props.userId}
@@ -213,6 +216,7 @@ class NotificationDropdownInner extends React.Component<PropsInner, State> {
                 }
                 analyticsLocation={this.props.analyticsLocation}
               />
+              {smartRender(this.props.Footer, {}, null)}
             </DropdownPanel>
           )}
         </div>
```

**Comment by sheng-duetto** [comment]
cc @jaapbakker88 @JelteF what do you guys think?

**Comment by jaapbakker88** [comment]
Sorry for the late response @sheng-duetto. I just released version `0.9.14` which includes an optional `Header` and `Footer` prop on the NotificationDropdown. 

You can see it in action here: 
https://getstream.github.io/react-activity-feed/#!/NotificationDropdown/1

I'm going to close this now but feel free to reach out if you run into any other issues.

**Comment by sheng-duetto** [comment]
@jaapbakker88 awesome TYVM confirmed it's working as expected!
